04_setup_ironic.sh

```diff
@@ -158,14 +158,6 @@ if [ "$NODES_PLATFORM" = "libvirt" ]; then
     fi
 fi
 
-
-# Wait for the downloader containers to finish, if they are updating an existing cache
-# the checks below will pass because old data exists
-sudo podman wait -i 1000 ipa-downloader
```

Comment:
I discussed removing this a while ago, but IIRC some folks (@derekhiggins @yrobla and others doing IPA testing) were still using it sometimes.

Since then we added ways to more easily customize the downstream image, but I wonder if we should consider making a conditional for this (defaulted to off) vs removing it completely, maybe it should be conditional on the `UPSTREAM_IRONIC` variable perhaps?

Reply:
Ya, it would be nice to keep it if we could, I haven't used it in a while but I could see myself wanting it again in the future.